src/dhtnode/request/neo/RemoveChannel.d

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+
+    RemoveChannel request implementation.
+
+    copyright:
+        Copyright (c) 2017 sociomantic labs GmbH. All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module dhtnode.request.neo.RemoveChannel;
+
+import dhtproto.node.neo.request.RemoveChannel;
+
+import dhtnode.connection.neo.SharedResources;
+import dhtnode.storage.StorageEngine;
+
+import swarm.neo.node.RequestOnConn;
+import swarm.neo.request.Command;
+
+import ocean.transition;
+import ocean.core.TypeConvert : castFrom, downcast;
+import ocean.core.Verify;
+
+/*******************************************************************************
+
+    DHT node implementation of the v0 RemoveChannel request protocol.
+
+*******************************************************************************/
+
+public scope class RemoveChannelImpl_v0 : RemoveChannelProtocol_v0
+{
+    /***************************************************************************
+
+        Checks whether the specified client is permitted to remove channels.
+
+        Params:
+            client_name = name of client requesting channel removal
+
+        Returns:
+            true if the client is permitted to remove channels
+
+    ***************************************************************************/
+
+    override protected bool clientPermitted ( cstring client_name )
+    {
+        return client_name == "admin";
```

Comment:
Small suggestion: I suggestion making `"admin"` a class constant, instead of inlining it here.